template.yml

```diff
@@ -64,13 +64,28 @@ Resources:
               - cloudwatch:DescribeAlarms
             Resource: "*"
 
+  ApiGatewayPermissions:
+    Type: AWS::IAM::Policy
+    Properties:
+      PolicyName: !Sub ApiGatewayPermissions-${AWS::StackName}
+      Roles:
+        - !Ref LambdaRole
+      PolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+          - Effect: Allow
+            Action:
+              - Apigateway:GET
+            Resource: "*"
```

Comment:
not sure what endpoint this is needed for, but could it be that you could limit it to  'arn:aws:apigateway:*::/restapis' without any wildcard? as far as I understand, this give get access to all our apis (I think?) If you also need 'arn:aws:apigateway:<region>::/restapis/<api-id-1>, 2,3....' then I guess it's more tricky

Reply:
probably ok as we need invoke to actually invoke